state/indexer/block/kv/kv_test.go

```diff
@@ -249,26 +249,14 @@ func TestBlockIndexerMulti(t *testing.T) {
 			q:       query.MustParse("block.height = 1"),
 			results: []int64{1},
 		},
-		"query return all events from a height - exact - no match.events": {
```

Comment:
These are just duplicate tests, piggy backing their removal onto this PR. We already did this for 0.38 and main